docs/source/fitting/fitfunctions/ChudleyElliotSQE.rst

```diff
@@ -0,0 +1,48 @@
+.. _func-Chudley-ElliotSQE:
+
+================
+ChudleyElliotSQE
+================
+
+.. index:: ChudleyElliotSQE
+
+Description
+-----------
+
+This fitting function models the dynamic structure factor
+for a particle undergoing jump diffusion using the Chudley-Elliot Jump diffusion model [1]_.
+
+.. math::
+
+   S(Q,E) = Height \cdot \frac{1}{\pi} \frac{\Gamma}{\Gamma^2+(E-Centre)^2}
+
+   \Gamma = \frac{\hbar}{\tau} \cdot \left(1 - \frac{sin(Ql)}{Ql}\right)
+
+where:
+
+-  :math:`Height` - Intensity scaling, a fit parameter
+-  :math:`Centre` - Centre of peak, a fit parameter
+-  :math:`Tau` - Residence time, a fit parameter
```

Comment:
Probably it is better to replace :math:`Tau` with :math:`\tau` so that the parameter looks the same as in the equation above

Reply:
I decided to use `Tau` becasue that is what the variable is called when assigning it in a function string.